services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowPayloadToFlowConverter.java

```diff
@@ -80,14 +81,15 @@ public static FlowPayload buildFlowPayloadByFlow(Flow flow) {
     }
 
     /**
-     * Builds {@link FlowPayload} instance by {@link Flow} instance.
+     * Builds {@link FlowPayload} instance by {@link ImmutablePair} instance.
      *
      * @param flowId flow id
-     * @param path {@link PathInfoData} instance
+     * @param pair {@link ImmutablePair} with paths
      * @return {@link FlowPayload} instance
      */
-    public static FlowPathPayload buildFlowPathPayloadByFlowPath(String flowId, PathInfoData path) {
-        return new FlowPathPayload(flowId, path);
+    public static FlowPathPayload buildFlowPathPayloadByFlowPath(
```

Comment:
Use/extend org.openkilda.northbound.converter.FlowMapper instead of this object.

Reply:
MapStruct requires setters in target class. So, we can't use lombok's @Value in target class anymore and must replace it with @Data. Is it ok?